build/cqs.tsv

```diff
@@ -166,6 +167,7 @@ org.scalatra:scalatra_2.11	2.3.0	compile
 org.scalatra:scalatra-auth_2.11	2.3.0	compile
 org.scalatra:scalatra-common_2.11	2.3.0	compile
 org.scalatra:scalatra-json_2.11	2.3.0	compile
+org.scalatra:scalatra-swagger_2.11	2.3.0	compile
```

Comment:
looks like our current CQ will cover this: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=8985